src/pages/Board/component/ItemTable.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import {useTranslation} from 'react-i18next'
+import Item from '../../../models/Item'
+import ModelState from '../../../models/bases/ModelState'
+import ErrorText from '../../../components/ErrorText'
+import Table from '../../../components/Table'
+import moment from 'moment'
+import {DATE_FORMAT} from '../../../constant'
+
+interface Props {
+	items: ModelState<Item[]>
+}
+
+const ItemTable: React.FunctionComponent<Props> = ({items}) => {
+	const [t] = useTranslation(['board', 'common'])
+	const {data, status, error} = items
+
+	const getTableColumns = () => [
+		{
+			title: t('date'),
+			dataIndex: 'date',
+			editable: true,
+			render: date => moment(date).format(DATE_FORMAT),
+		},
+		{
+			title: t('item'),
+			dataIndex: 'name',
+			editable: true,
+		},
+		{
+			title: t('price'),
+			dataIndex: 'price',
+			editable: true,
+		},
+		{
+			title: t('quantity'),
+			dataIndex: 'quantity',
+			editable: true,
+		},
+		{
+			title: t('total'),
+			dataIndex: 'total',
+		},
+		{
+			title: t('category'),
+			dataIndex: 'category.name',
+			editable: true,
+		},
+		{
+			title: t('note'),
+			dataIndex: 'note',
+			editable: true,
+		},
+	]
+
+	const renderContent = () => {
+		if (status === 'error') {
+			return <ErrorText>{error}</ErrorText>
+		}
+
+		return (
+			<Table
+				columns={getTableColumns()}
```

Comment:
I have a mixed feeling on this way. If we called this `getTableColumns()` to get columns. `Columns` prop of the Table will change everytime this `ItemTable` component re-render. Which leads to the Table re-renders itself. I think we can prevent this by just passing a `column` table here.
Essentially:
```
const columns = [{title: '', dataIndex: '', editable: true}...]
```

Reply:
May be I need to change how columns is get in Category tables as well